x-pack/test/functional_enterprise_search/page_objects/app_search.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FtrProviderContext } from '../ftr_provider_context';
+import { TestSubjects } from '../../../../../test/functional/services/test_subjects';
+
+export function AppSearchPageProvider({ getService, getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects(['common']);
+  const testSubjects = getService('testSubjects') as TestSubjects;
+
+  return {
+    async navigateToPage() {
+      return await PageObjects.common.navigateToApp('app_search');
+    },
+
+    async getEngineLinks() {
+      const engines = await testSubjects.find('appSearchEngines');
+      return await testSubjects.findAllDescendant('engineNameLink', engines);
+    },
+
+    async getMeatEngineLinks() {
```

Comment:
Ahahaha I'm so sorry, this cracked me up but
```suggestion
    async getMetaEngineLinks() {
```
Now I'm thinking real hard about what a meat engine would look like 😂

Reply:
That was totally intentional 🍖 .